packages/manager/apps/web-hosting/src/pages/websites/Websites.page.tsx

```diff
@@ -399,6 +401,10 @@ export default function Websites() {
             <OdsPopover
```

Comment:
I don't understand why it's so complex to use.

I think the implemention is bad ( May be since migration ?) : https://ovh.github.io/design-system/latest/?path=/docs/ods-components-popover--migration-from-17-x

May be fix the implementation can fix your problem ?